tests/dm_syncer/conf/dm-worker1.toml

```diff
@@ -1,13 +1,2 @@
-# Worker Configuration.
-
-source-id = "mysql-replica-01"
-flavor = ""
-enable-gtid = false
-relay-binlog-name = ""
-relay-binlog-gtid = ""
-
-[from]
-host = "127.0.0.1"
-user = "root"
-password = ""
-port = 3306
```

Comment:
_dm-syncer_ may still want to do this in itself. 😂 
cc @WangXiangUSTC

Reply:
`dm-syncer`'s test only use dm-worker to load full data, we can remove these code